apps/meteor/client/views/room/HeaderV2/Omnichannel/OmnichannelRoomHeader.tsx

```diff
@@ -44,9 +45,10 @@ const OmnichannelRoomHeader = ({ slots: parentSlot }: OmnichannelRoomHeaderProps
 					<BackButton routeName={currentRouteName} />
 				</HeaderToolbar>
 			),
+			insideContent: <OmnichannelRoomHeaderTag room={room} />,
 			posContent: <QuickActions />,
 		}),
-		[isMobile, currentRouteName, parentSlot],
+		[isMobile, currentRouteName, parentSlot, room],
```

Comment:
Is `room` a stable object (i.e memoized)? I think this memo will recompute everytime...